.github/scripts/pr_body_check.py

```diff
@@ -0,0 +1,63 @@
+import os
+import re
+
+def warn(msg: str) -> None:
+    print(f"::warning::{msg}")
+
+def normalize(s: str) -> str:
+    return s.replace('\r\n', '\n').replace('\r', '\n').strip()
+
+def extract_sections(text: str) -> dict[str, str]:
+    # Match lines like: ### 🧑‍🔬 Unit Testing
+    section_regex = re.compile(
+        r'^\s*#{2,6}\s*([\W_]{1,4} [^\n]+)\n([\s\S]*?)(?=^\s*#{2,6}\s*[\W_]{1,4} [^\n]+\n|$)',
+        re.MULTILINE,
+    )
+    return {m.group(1).strip(): m.group(2).strip() for m in section_regex.finditer(text)}
+
+def has_checked_box(text: str) -> bool:
+    # Looks for - [x] or - [X] at the start of a line
+    return bool(re.search(r'^\s*-\s*\[[xX]\]', text, re.MULTILINE))
```

Comment:
I often use "N/A" for this section of the PR description when I only update documentation, or something that doesn't require unit testing. Should we modify this to support "N/A"? I suppose the same question would apply to other sections, as well...